modules/configuration/tools/import_projects.php

```diff
@@ -12,32 +21,64 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://www.github.com/aces/Loris/
  */
-
 require_once __DIR__ . "/../../../vendor/autoload.php";
 $client = new NDB_Client();
 $client->makeCommandLine();
 $client->initialize();
+$factory   = NDB_Factory::singleton();
+$config    = $factory->config(__DIR__ . "/../../../project/config.xml");
+$subprojs  = $config->getSettingFromXML("subprojects");
+$db        = $factory->database();
+$optionpos = 1; //The position of the option in the command line.
 
-$factory  = NDB_Factory::singleton();
-$config   = $factory->config(__DIR__ . "/../../../project/config.xml");
-$subprojs = $config->getSettingFromXML("subprojects");
-$db       = $factory->database();
-foreach ($subprojs['subproject'] as $row) {
-    $windowDiff = "optimal";
-    if (isset($row['options']) && isset($row['options']['WindowDifference'])) {
-        $windowDiff = $row['options']['WindowDifference'];
+if (is_null($argv[$optionpos])
+) {
+    echo ("The script needs an argument. The arguments are -s,-p or -a.\r\n");
+    echo ("-s -> Imports only the subprojects from the XML.\r\n");
+    echo ("-p -> Imports only the projects from the XML.\r\n");
+    echo ("-a -> Imports both the subprojects and the projects \r\n");
+    echo ("      from the XML.\r\n");
+    exit (2);
+}
+if ((isset($argv[$optionpos]) && $argv[$optionpos] === "-s")
+    || (isset($argv[$optionpos]) && $argv[$optionpos] === "-a")
+) {
+    foreach ($subprojs['subproject'] as $row) {
+        $windowDiff = "optimal";
+        if (isset($row['options']) && isset($row['options']['WindowDifference'])) {
+            $windowDiff = $row['options']['WindowDifference'];
+        }
+        $ins = array(
+                'SubprojectID'     => $row['id'],
+                'title'            => $row['title'],
+                'useEDC'           => 0,
+                'WindowDifference' => $row['options']['WindowDifference'],
```

Comment:
I think this line should be "WindowDifference" => $windowDiff to fix the problem @taracampbell is having.